coalib/misc/Exceptions.py

```diff
@@ -15,6 +17,12 @@ def get_exitcode(exception, log_printer=None):
         exitcode = 0
     elif isinstance(exception, SystemExit):
         exitcode = exception.code
+    elif (isinstance(exception, ContextualVersionConflict) or
+          isinstance(exception, VersionConflict)):
+        log_message = Constants.VERSION_CONFLICT_MESSAGE
+        log_message += exception.report()
+        log_printer.log_exception(log_message, exception)
+        exitcode = 4
```

Comment:
this one's a new exitcode IMO definitely and needs an own line of docs, not that it takes long to write it does it?